homeassistant/components/glances/sensor.py

```diff
@@ -256,218 +263,199 @@ def _migrate_old_unique_ids(
                         hass,
-                        f"{client.host}-{name} {sensor['label']} {description.name_suffix}",
+                        f"{coordinator.host}-{name} {sensor['label']} {description.name_suffix}",
                         f"{sensor['label']}-{description.key}",
                     )
-                    dev.append(
+                    entities.append(
                         GlancesSensor(
-                            client,
+                            coordinator,
                             name,
                             sensor["label"],
                             description,
                         )
                     )
         elif description.type == "raid":
-            for raid_device in client.api.data[description.type]:
+            for raid_device in coordinator.data[description.type]:
                 _migrate_old_unique_ids(
                     hass,
-                    f"{client.host}-{name} {raid_device} {description.name_suffix}",
+                    f"{coordinator.host}-{name} {raid_device} {description.name_suffix}",
                     f"{raid_device}-{description.key}",
                 )
-                dev.append(GlancesSensor(client, name, raid_device, description))
-        elif client.api.data[description.type]:
+                entities.append(
+                    GlancesSensor(coordinator, name, raid_device, description)
+                )
+        elif coordinator.data[description.type]:
             _migrate_old_unique_ids(
                 hass,
-                f"{client.host}-{name}  {description.name_suffix}",
+                f"{coordinator.host}-{name}  {description.name_suffix}",
                 f"-{description.key}",
             )
-            dev.append(
+            entities.append(
                 GlancesSensor(
-                    client,
+                    coordinator,
                     name,
                     "",
                     description,
                 )
             )
 
-    async_add_entities(dev, True)
+    async_add_entities(entities)
 
 
-class GlancesSensor(SensorEntity):
+class GlancesSensor(CoordinatorEntity[GlancesDataUpdateCoordinator], SensorEntity):
     """Implementation of a Glances sensor."""
 
     entity_description: GlancesSensorEntityDescription
     _attr_has_entity_name = True
-    _attr_should_poll = False
 
     def __init__(
         self,
-        glances_data: GlancesData,
+        coordinator: GlancesDataUpdateCoordinator,
         name: str | None,
         sensor_name_prefix: str,
         description: GlancesSensorEntityDescription,
     ) -> None:
         """Initialize the sensor."""
-        self.glances_data = glances_data
+        super().__init__(coordinator)
         self._sensor_name_prefix = sensor_name_prefix
-        self.unsub_update: CALLBACK_TYPE | None = None
-
         self.entity_description = description
         self._attr_name = f"{sensor_name_prefix} {description.name_suffix}"
```

Comment:
`sensor_name_prefix` looks like it can be an empty string. There will be a whitespace before the suffix in the name here in that case.

Reply:
I can add `.strip()` to remove the whitespace. Would that cause any breaking change for existing instances when the user updates?

Reply:
It won't cause a breaking change as a unique id is set already in a previous release. The entity id won't change and only a default name would change.